### ISSUE TITLE
Moving binary files doesn't work

### ISSUE THREAD

**Issue by jdm** [servo-automation/upstream-wpt-sync-webhook#9]
Moving a binary file within web-platform-tests yields a diff made of an addition and a removal, and git apply fails with:
```
Checking patch 2dcontext/2x2-new.png...
error: cannot apply binary patch to '2dcontext/2x2-new.png' without full index line
error: 2dcontext/2x2-new.png: patch does not apply
Checking patch 2dcontext/2x2.png...
error: cannot apply binary patch to '2dcontext/2x2.png' without full index line
```
```diff
diff --git a/2dcontext/2x2-new.png b/2dcontext/2x2-new.png
new file mode 100644
index 0000000..9be3156
Binary files /dev/null and b/2dcontext/2x2-new.png differ
diff --git a/2dcontext/2x2.png b/2dcontext/2x2.png
deleted file mode 100644
index 9be3156..0000000
Binary files a/2dcontext/2x2.png and /dev/null differ
```

**Comment by jdm** [comment]
This requires --binary to be added to the git show command.

**Comment by jdm** [comment]
Fixed by e9c974ee3b8797de388e106abdaea53a75160b11.
